routes/userRouteNotes.js

```diff
@@ -0,0 +1,57 @@
+router.get('/widgets/:widgetId', ( request, response, next ) => {
+  const { widgetId } = request.params
+
+  User.find( {'widgets._id': widgetId},function(error, widgets){
+    return widgets
+  })
+  .then( data => console.log('finished') )
+})
+
+router.post('/dashboards/:id/widgets', (request, response, next ) => {
+  Widget.create(request.body, (error, post) => {
```

Comment:
Please consider a different name for the `post` parameter - this is not an intention revealing name